### ISSUE TITLE
Unable to use Google OAuth in production

### ISSUE THREAD

**Issue by abacaj** [supabase/supabase#4124]
# Bug report

In reference to: https://github.com/supabase/supabase/issues/2128

I received the same issue from google OAuth consent approval.
I did not ask for any sensitive or restricted scopes, but it is asking me to **verify I own the supabase.io domain.**

![image](https://user-images.githubusercontent.com/7272343/143812139-d989dd78-49db-4189-8f29-8c10e38d9807.png)

![image](https://user-images.githubusercontent.com/7272343/143812013-bb41339c-38d5-46a7-b440-26312701199b.png)

**Comment by liaujianjie** [comment]
Just FYI, adding an app logo will also trigger this verification. Did you do that?

In any case, all you have to do is to reply to that email saying that you are using a 3rd party service, Supabase, for authenticating your users. I did that and it worked for me.

**Comment by abacaj** [comment]
@liaujianjie yes that's right I added a app logo as part of the consent screen flow. Will try that thank you.

**Comment by tourbillonlabs** [comment]
If you get knocked back after the initial justification, I've just gotten my app verified and documented my experience in this comment: https://github.com/supabase/supabase/discussions/2925#discussioncomment-1716295

Hope it helps. Good luck!

**Comment by kiwicopple** [comment]
I'll convert this to a discussion because it looks like useful knowledge for others!
